source/D2Lang/include/D2Unicode.h

```diff
@@ -68,4 +68,13 @@ struct D2LANG_DLL_DECL Unicode {
 
   // D2Lang.0x6FC11C50 (#10023) ?directionality@Unicode@@QAE?AW4Direction@1@XZ
   Direction directionality();
+
+  /**
+   * Returns whether or not this Unicode character is in 
+   * "C0 Controls and Basic Latin" character block, otherwise known as
+   * the ASCII character block.
+   *
+   * D2Lang.0x6FC11080 (#10024) ?isASCII@Unicode@@QBEHXZ
+   */
+  int isASCII() const;
```

Comment:
Could you replace the `int` by `BOOL` (not `bool`) ? You'll need to add  `#include <D2BasicTypes.h` at the top of the header

Reply:
Done.